src/data-table/data-table.js

```diff
@@ -587,24 +570,63 @@ export function Unstable_DataTable(props: DataTablePropsT) {
     [props.onSort],
   );
 
+  const [columnHighlightIndex, setColumnHighlightIndex] = React.useState(-1);
+  const [rowHighlightIndex, setRowHighlightIndex] = React.useState(-1);
+
+  function handleRowHighlightIndexChange(nextIndex) {
+    setRowHighlightIndex(nextIndex);
+    if (gridRef.current) {
+      if (nextIndex >= 0) {
+        // $FlowFixMe - unable to get react-window types
+        gridRef.current.scrollToItem({rowIndex: nextIndex});
+      }
+      if (props.onRowHighlightChange) {
+        props.onRowHighlightChange(nextIndex, rows[nextIndex - 1]);
+      }
+    }
+  }
+
+  const handleRowMouseEnter = React.useCallback(
```

Comment:
is there any need for a `handleRowMouseLeave` handler?

Reply:
yeah it might be worth exploring how to 'unhighlight' the row when mouse leaves the whole table. right now the last highlighted row stays